AndyDorokhin/HW10/extras/adam_eve.py

```diff
@@ -0,0 +1,30 @@
+""" 
+    According to the creation myths of the Abrahamic religions, 
+    Adam and Eve were the first Humans to wander the Earth.
+
+    You have to do God's job. The creation method must return an array of length 2 containing objects (representing Adam and Eve). The first object in the array should be an instance of the class Man. The second should be an instance of the class Woman. 
+    Both objects have to be subclasses of Human. 
+    Your job is to implement the Human, Man and Woman classes.
+"""
+class Human:
```

Comment:
good job
look at this version of the code

def God():
		return (Man(), Woman())
	
	
class Human:
	pass
	
	
class Man(Human):
	pass
	
	
class Woman(Human):
	pass